ui/src/app/components/side-nav.tsx

```diff
@@ -257,7 +259,7 @@ export class SideNav extends React.Component<SideNavProps, SideNavState> {
   }
 
   render() {
-    const {profile} = this.props;
+    const {profile, authStore: {isSignedIn}} = this.props;
```

Comment:
This entire component requires that you are signed in currently, per the required `profile` property. Perhaps naming or documentation for the component should make this clearer, but I think adding the direct authStore integration is probably misleading (many errors would be thrown from this method if you are not signed in)

Reply:
Thanks - I removed the authstore completely and just have the profile enabled. The profile page is still guarded on sign in, so there should be no need for a logic check.

Thanks!